reaktive/src/commonMain/kotlin/com/badoo/reaktive/single/FlatMapObservable.kt

```diff
@@ -2,6 +2,15 @@ package com.badoo.reaktive.single
 
 import com.badoo.reaktive.observable.Observable
 import com.badoo.reaktive.observable.flatMap
+import com.badoo.reaktive.observable.map
 
 fun <T, R> Single<T>.flatMapObservable(mapper: (T) -> Observable<R>): Observable<R> =
-    asObservable().flatMap(mapper)
\ No newline at end of file
+    asObservable().flatMap(mapper)
+
+fun <T, U, R> Single<T>.flatMapObservable(
```

Comment:
Please choose one style, I would put whole definition on one line if it fits. Now some are multi line and some are single line. Max line width is 140

Reply:
Fixed